docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -1,387 +1,390 @@
+3. [Install the Micro-RDK](#install-the-micro-rdk)
+
+### Hardware Requirements
+
+You need an Espressif ESP32 microcontroller to use the micro-RDK.
+Viam recommends purchasing the ESP32 with a [development board](https://www.espressif.com/en/products/devkits).
+
+* **Minimal configuration:** 384kB Ram 4MB flash
+* **Recommended configuration:** 384kB Ram + 8MB SPIRAM + 4MB Flash
+
+### Software Requirements
+
+The micro-RDK is written in Rust.
+To be able to program the ESP32 on macOS and Linux systems, you must install the following software on your development machine:
+
+#### Install Dependencies
+
+{{< tabs >}}
+{{% tab name="Linux (Ubuntu)" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+sudo apt-get install git libssl-dev dfu-util libusb-1.0-0 libudev-dev
+```
+
+{{% /tab %}}
+{{% tab name="macOS" %}}
+
+If you haven't already, install Homebrew:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+Then, install `dfu-util`:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+brew install dfu-util
+```
+
+{{% /tab %}}
+{{% /tabs %}}
+
+#### Install `Rust`
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+See the [Rust Installation guide](https://www.rust-lang.org/tools/install) for more information and other installation methods.
+
+#### Install `cargo-generate` with `cargo`
+
+ `cargo` installs automatically when downloading Rust with rustup.
+
+Run the following command to install `cargo-generate`:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+cargo install cargo-generate
+```
+
+#### Install `espflash`
+
+Run the following command to install `espflash`
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+cargo install espflash
+```
+
+## Set up your development enviroment
+
+{{< tabs >}}
+{{% tab name="Canon CLI (recommended)" %}}
+
+### Use Viam's Canon CLI utility
+
+[Canon](https://github.com/viamrobotics/canon) is a CLI utility for managing a Docker-based canonical environment.
+
+Canon requires a working installation of Docker Engine.
+
+[Install Docker Engine](https://docs.docker.com/engine/install/) before proceeding.
+If you are running Docker Engine on Linux, make sure that you go through the [post installation steps](https://docs.docker.com/engine/install/linux-postinstall/) to run Docker automatically on startup.
+
+{{< tabs >}}
+{{% tab name="Linux (Ubuntu)" %}}
+
+Make sure your system has [Go 1.19](https://golangtutorial.dev/news/go-1.19-version-released/#major-changes-in-go-119-version) or later installed.
+Verify your version of Go with `go version`.
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+go install github.com/viamrobotics/canon@latest
```

Comment:
The homebrew install method (brew install canon) works on linux as well as MacOS. Likewise, the "go install ..." method here works on MacOS too. Why show different install instructions between OSes?

Reply:
So the other changes here look fine by me. I still think it's odd to have different install methods. Maybe the tabs should be "Direct Go Install (Linux/MacOS)" and "Homebrew (Linux/MacOS)" since either method works for either OS.

Reply:
Given that each linux distribution has its own package manager I don't want users to be forced to use Homebrew. Installing go via the command line seems the least invasive and relatively easy to revert if something goes wrong, the user would have to figure out how to install golang for their particular distro

Reply:
Yeah, I'm not saying remove either, just relabel them. Let the user's choose which method they prefer. If you already have homebrew installed, it's easier than trying to figure out how to install go. If you already have go, it's easier than trying to install homebrew. :-) But as noted, not blocking on this either way.

Reply:
All good now that I see your latest changes. Sorry for the cross talk. I should've just waited 10 more minutes to reply. :-)